aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move

```diff
@@ -0,0 +1,48 @@
+/// This module provides an interface for aggregators (version 2).
+/// Only skeleton - for AggregagtorSnapshot - is provided at this time,
+/// to allow transition of usages.
+module aptos_framework::aggregator_v2 {
+    use std::string::String;
+
```

Comment:
We return an error if the elements is wrong: we should define what these errors are here? i.e.
```
const EINVALID_SNAPSHOT_TYPE: u64 = ...
```

Reply:
Done.